jina/peapods/runtimes/remote/jinad/__init__.py

```diff
@@ -0,0 +1,40 @@
+from argparse import Namespace
+from typing import Union, Dict, Optional
+
+from jina.peapods.runtimes.remote.jinad.api import PeaAPI
+from jina.peapods.runtimes.remote import RemoteRuntime
+from jina.helper import cached_property, namespace_to_dict, colored, typename
+from .api import get_jinad_api
+
+
+class RemoteJinaDRuntime(RemoteRuntime):
```

Comment:
Or `JinadRuntime` because `SSH` and `Jinad` are already in the `runtimes/remote` folder
```suggestion
class JinadRemoteRuntime(BaseRemoteRuntime):
```

Reply:
Use `JinadRemoteRuntime` to be consistent with `JinadAPI`